Guesser/DoctrineFieldGuesser.php

```diff
@@ -62,7 +62,8 @@ public function getAllFields($class)
     public function getManyToMany($model, $fieldPath)
     {
         $resolved = $this->resolveRelatedField($model, $fieldPath);
-        return !$this->getMetadatas($resolved['class'])->isAssociationWithSingleJoinColumn($resolved['field']);
+        $metadata = $this->getMetadatas($resolved['class']);
+        return $metadata->hasAssociation($resolved['field']) && !$metadata->isAssociationWithSingleJoinColumn($resolved['field']);
```

Comment:
Bug fixes can also be BC-breaking :wink: But I see the problem now, thanks for the fix!